src/app/logic/processGame.js

```diff
@@ -0,0 +1,146 @@
+import Answers from '../components/Answers';
+import VisualImage from '../components/VisualImage';
+import { getQuestion, initGame, isGameInitialized } from './quiz';
+
+class Game {
+  getTimeLeft() {
+    return this.timeLeft;
+  }
+
+  getMode() {
+    return this.mode;
+  }
+
+  getRunning() {
+    return this.running;
+  }
+
+  getQuestions() {
+    return this.questions;
+  }
+
+  reduceTime() {
+    this.timeLeft -= 0.5;
+  }
+
+  addQuestion(question) {
+    this.questions.push(question);
+  }
+
+  newGame(mode) {
+    this.timeLeft = 5 || process.env.QUIZ_MAX_TIME_SECONDS;
+    this.questions = [];
+    this.mode = mode;
+    this.running = false;
```

Comment:
Is there any reason to add 'return this' for chaining sake?

Reply:
No, game is used in global scope, so any reference is unnecessary

Reply:
Moved function to the top, changed name to initGame